CreditManagement/querysets.py

```diff
@@ -0,0 +1,280 @@
+from django.db import models
+from django.db.models import Q, F, Value, Sum, OuterRef, Subquery, ExpressionWrapper, Count
+from django.db.models.functions import Coalesce, Cast
+
+from Dining.models import DiningEntry, DiningList
+from UserDetails.models import User, Association
+
+
+class AbstractTransactionQuerySet(models.QuerySet):
+    """
+    An abstract Transaction Queryset, defines base methods for filtering and balance computations
+    For any model inheriting the AbstractTransaction class
+    """
+
+    def __filter_for__(self, item, source_column, target_column):
```

Comment:
Use for private methods and functions the naming convention _name (one underscore at the beginning). (https://docs.python.org/3/tutorial/classes.html#private-variables)

Reply:
Oops, ik ga dit fixen

Reply:
It's like I said:

> _name (one underscore at the beginning).

Not `_name_`. I fixed it now in this branch, so you can pull these.